test/unit/models/classes/Security/DataAccess/Repository/PlatformJwksRepositoryTest.php

```diff
@@ -63,7 +63,7 @@ public function setUp(): void
     public function testFind(): void
     {
         $keyChain = new KeyChain('id', 'name', new Key('123456'), new Key('654321'));
-        $collection = new KeyChainCollection(...[$keyChain]);
+        $collection = new KeyChainCollection([$keyChain]);
```

Comment:
**question**: what if we leave a variable-argument list for the collection? Can we just pass a single argument here without any array wrapping?